solution/ui/e2e/cypress/e2e/custom-login-page.cy.js

```diff
@@ -0,0 +1,43 @@
+describe("custom login page", { scrollBehavior: "center" }, () => {
+    beforeEach(() => {
+        cy.clearIndexedDB();
+        cy.intercept("/**", (req) => {
+            req.headers["x-automated-test"] = Cypress.env("DEPLOYING");
+        });
+    });
+
+    it("has a flash banner at the top with a link to a feedback survey", () => {
+        cy.viewport("macbook-15");
+        cy.visit("/login");
+        cy.checkFlashBanner();
```

Comment:
this test suite needs to check for a11y issues and it needs to run the `checkLinkRel` method to ensure links are properly formed.  

Take a look at the very first test in the Error Page Test suite and you can see `injectAxe`, `checkAccessibility`, and `checkLinkRel` all being used.  Just copy how its done there into one of the tests here in `custom-login-page`.